templates/zerver/help/set-default-streams-for-new-users.md

```diff
@@ -0,0 +1,23 @@
+# Set default streams for new users
+
+Default streams for an organisation is set by an administrator.
+
+Follow the step given below to set the default streams for new users using an administrator account:
+
+* Click on the cog (![cog](/static/images/help/cog.png)) icon in the upper right corner of the sidebar.
+
+* Choose the **Administration** option in the panel shown below:
+
+![admin](/static/images/help/admin.png)
+
+* The **Administration** window will appear like the one below:
+
+![admin window](/static/images/help/admin-window.png)
+
+* Click on the **Default Streams** option as shown below:
+
+![default streams](/static/images/help/default-streams.png)
+
+* To delete a stream from **Default Streams** click on the **Remove from default** button next to the stream's name that you want to remove.
+
+* To add a stream to **Default Streams**, Enter the name of stream in the box next to **Stream Name** and press the enter key.
```

Comment:
I think description for adding a stream should be first, before removing a stream from `Default Streams`